service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/PolicyEvaluationCache.java

```diff
@@ -29,24 +29,26 @@
 
     void set(PolicyEvaluationRequestCacheKey key, PolicyEvaluationResult value);
 
-    void setResourceTranslation(String zoneId, String fromResourceId, String toResourceId);
-
-    void setResourceTranslations(String zoneId, Set<String> fromResourceIds, String toResourceId);
-
     void reset();
 
     void reset(PolicyEvaluationRequestCacheKey key);
 
     void resetForPolicySet(String zoneId, String policySetId);
 
+    void setPolicySetIfNotExists(String zoneId, String policySetId);
```

Comment:
setPolicySetIfNotExists, setResourceIfNotExists, setSubjectIfNotExists do not need to be part of cache interface.  These methods are implemented and used by AbstractPolicyEvaluationCache only.  However, setIfNotExists should be defined by interface and implemented by each cache implementation.